requirements.txt

```diff
@@ -18,3 +18,77 @@ python-dateutil==2.6.0 \
 s3transfer==0.1.10 \
     --hash=sha256:bc52f38637f37572ae180c08a2dd87a4825b46713797633aab6e1a42d6b2b8ff \
     --hash=sha256:ba1a9104939b7c0331dc4dd234d79afeed8b66edce77bbeeecd4f56de74a0fc1
+python-decouple==3.0 \
```

Comment:
@akatsoulas @andrewkrug I am not sure what's the proper way to proceed with the project dependencies

* ``setup.py`` install requires are better to package cis
* ``requirements.txt`` supports hash checking to make sure that allow more fine grained integrity across envs (also improves deployment security by making the env reproducible).

thoughts ?